src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,142 @@
+package duke;
+
+import java.util.Scanner;
+
+import duke.tasktype.Deadline;
+import duke.tasktype.Event;
+import duke.tasktype.Todo;
+import duke.exception.WrongCommandFormatException;
+
+/**
+ * Class to handle all the user inputs.
+ *
+ * @author Houten Teo
+ * @version CS2103T week 3
+ */
+public class Parser {
```

Comment:
great to see that you placed the middleware logic into a Parser file. Exceptional quality of work that shows foresight into how you considered the project structure might become and this ensures that your project is modular, where people can easily contribute to your project using existing code.